code/game/objects/structures/artstuff.dm

```diff
@@ -185,6 +185,8 @@
 
 /obj/item/canvas/proc/try_rename(mob/user)
 	var/new_name = stripped_input(user,"What do you want to name the painting?")
+	if(!new_name)
+		new_name = "Untitled Artwork"
 	if(new_name != painting_name && new_name && user.canUseTopic(src,BE_CLOSE))
```

Comment:
But it's checked here ?

Reply:
That doesn't stop you from hanging it with no name which creates the error, if I'm understanding it right. This is actually setting a name for when there's no name.

Reply:
There's a default name set above.

Reply:
Yeah, but doesn't using the input change that?

Reply:
Removed that, put it in the save and load. This should prevent new NULL names from popping up, and get rid of the old ones.